peachjam/models/generic_document.py

```diff
@@ -85,6 +86,23 @@ class Meta(CoreDocument.Meta):
     def __str__(self):
         return self.title
 
+    def apply_labels(self):
+        # label to indicate that this legislation is repealed
+        label, _ = Label.objects.get_or_create(
+            code="repealed", defaults={"name": "Repealed", "code": "repealed"}
+        )
+        # apply label if repealed
+        if self.metadata_json.get("repeal"):
+            self.repealed = True
```

Comment:
This method doesn't need to be responsible for setting this field. Firstly because applying labels should be driven by this field (so setting it doesn't make sense) and secondly because this is done by the indigo ingestor.

So drop this line, and rely on `self.repealed` rather than `metadata_json`.

See adapters.py line 215 for where it is set.